os_net_config/sriov_bind_config.py

```diff
@@ -119,14 +125,15 @@ def bind_vfs(sriov_bind_pcis_map=None):
         for vf_pci in pcis_list:
             vf_pci_driver_path = _PCI_DRIVER_FILE_PATH % {"pci": vf_pci}
             if not os.path.exists(vf_pci_driver_path):
-                pci_driver_bind_file_path = _PCI_DRIVER_BIND_FILE_PATH %\
-                    {"driver": driver}
+                pci_driver_bind_file_path = _PCI_DRIVER_BIND_FILE_PATH % {
+                    "driver": driver
+                }
                 try:
-                    with open(pci_driver_bind_file_path, 'w') as f:
+                    with open(pci_driver_bind_file_path, "w") as f:
                         f.write("%s" % vf_pci)
```

Comment:
Why use `"%s" % vf_pci` here? If it is a string, just write it. Otherwise, just `write(str(vf_pci))`